index.html

```diff
@@ -19,220 +20,257 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Bezalell Portfolio</title>
-</head>
-<body>
+  </head>
+  <body>
+    <!-- navbar -->
     <header>
```

Comment:
- [ ] As requested by the previous reviewer, kindly fix all sections in your mobile version, the mobile version should still look as expected even after using media queries to build the desktop version. Kindly fix that please.
### Figma header
![image](https://user-images.githubusercontent.com/94551484/222881742-79177d90-32ba-4dc7-895e-815054a330ad.png)
### Your Header
![image](https://user-images.githubusercontent.com/94551484/222881863-b1e06de6-6907-43a5-9b26-3560fe94e561.png)

### FIgma Projects section
![image](https://user-images.githubusercontent.com/94551484/222881891-8b0ae020-417c-46d6-9fe9-17fa81c43459.png)
### Your project section
![image](https://user-images.githubusercontent.com/94551484/222881996-302d25f6-d6b6-474c-9c33-0f9a889110b6.png)
![image](https://user-images.githubusercontent.com/94551484/222882166-9bf4db42-9a2c-4931-a8e1-f4559845be20.png)

### About section Figma
![image](https://user-images.githubusercontent.com/94551484/222882165-6624e988-3336-4e5e-a8ab-558f44b26f5a.png)
### Your about section
![image](https://user-images.githubusercontent.com/94551484/222882299-37d60e02-797a-4c17-9541-c91a7850d52f.png)